avm/res/network/p2s-vpn-gateway/tests/e2e/defaults/main.test.bicep

```diff
@@ -0,0 +1,65 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults'
+metadata description = 'This instance deploys the module with the minimum set of required parameters.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-network.p2svpngateway-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param resourceLocation string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'vscmin'
```

Comment:
Beware, this serviceshort is already used by a different module (`avm\res\network\vpn-server-configuration\tests\e2e\defaults\main.test.bicep`). Is this a copy-paste error?
If not too long, I'd strongly recommend changing this short to `npvgmin`. The same comment applies to the other test files.

Reply:
Yup, copy & paste error, fixed as recommended